swig/cpp/src/Session.cpp

```diff
@@ -427,12 +427,14 @@ shared_ptr<Iter_Change> Subscribe::get_changes_iter(const char *xpath)
     }
 }
 
-shared_ptr<Operation> Subscribe::get_change_next(shared_ptr<Iter_Change> iter, shared_ptr<Val_Holder> new_value,\
-                                            shared_ptr<Val_Holder> old_value)
+shared_ptr<Operation> Subscribe::get_change_next(shared_ptr<Iter_Change> iter, shared_ptr<Val> new_value,\
```

Comment:
Yes, all output arguments in C api operation, new_val, old_val would be members of Change class.